server/homebrew.api.js

```diff
@@ -46,7 +37,7 @@ const newBrew = (req, res)=>{
 	}
 
 	brew.authors = (req.account) ? [req.account.username] : [];
-	brew.text = mergeBrewText(brew.text, brew.style);
+	brew.text = brewUtils.mergeBrewText(brew, { style: true });
```

Comment:
Is there ever a case where `style` *isn't* included? I think we want to just default to always including the style text. Same with `metadata`. I can't think of an instance where we want to update a brew without saving its metadata into the text or download a file without the metadata (especially if we are not including sensitive info like ids).